Classes/Database/Schema/SchemaUpdateType.php

```diff
@@ -72,6 +67,57 @@ class SchemaUpdateType extends Enumeration
     const TABLE_CLEAR = 'table.clear';
 
     /**
+     * All safe update types
+     */
+    const GROUP_SAFE = 'all.safe';
```

Comment:
This can be `SAFE`. It's obvious that this is a group since it doesn't have the `FIELD_` or `TABLE_` prefix.